src/Tracker/index.js

```diff
@@ -97,7 +98,7 @@ const formatProperties = properties => {
 }
 
 const userAnalyticsEvent = (eventName, properties) => {
-  dispatchEvent(new CustomEvent('userAnalyticsEvent', {detail: {eventName, properties}}));
+  dispatchEvent(new CustomEvent('userAnalyticsEvent', {detail: {eventName, properties, isCrossDevice: false}}));
```

Comment:
Is there a way to access the value of globals.mobileConnected(stored in redux)  in this function cleanly? If there is a way to do this, the boolean isCrossDevice can be set to the value of globals.mobileConnected and there is no need to override in Router/index.js

Reply:
You cannot easily access the redux store here. I can see that this function is called inside the `sendEvent` which is used inside the `StepsRouter` component. Inside the `StepsRouter` you could access the `mobileFlow` parameter (which tells you if the SDK is used as cross device client or not) from the store or from one of the `StepsRouter` prop. I'm not sure if that will do the trick though because sendEvent is called in different places within the code.

Reply:
@stefaniacardenas Thanks for sharing your thoughts. That was my concern too. I want the code changes to be very less intrusive. So, didn't take that approach.